### DISCUSSION TITLE
FloatingInfo

### DISCUSSION THREAD

**Discussion by M4rc09696** [DualSPHysics/DualSPHysics#280] [DUALSPHYSICS_V5.2]
Hello everyone,

I'm writing because I'm having an issue with a simulation in DualSPHysics. I tried to follow the YouTube tutorial "CaseWavesFloating." Everything goes well until the simulation stage, and even the animation in Paraview looks fine. However, when I try to extract the data for the floating part (using the "FloatingInfo" command in DesignSPHysics), I find very high velocity values (in addition to the time step being multiplied by 10^12). I've attached an image of the table and the "Case Summary."

Additionally, in another simulation using the "ComputeForces" command, I encountered force values in the order of 10^13. What could be the problem? Thank you very much for your help.
![Excel_FloatingInfo](https://github.com/user-attachments/assets/a28ba169-b375-4f6d-83e4-130a33a6f3ab)
![Define_Constants](https://github.com/user-attachments/assets/d672a3b3-1c03-4f58-8172-8b616d4f65c0)
![Execution_Parameters](https://github.com/user-attachments/assets/f7420375-e721-41b4-bf65-9dd5fbf7bd86)
![Case_Summary_1](https://github.com/user-attachments/assets/81e62212-9e1f-4de1-8a33-f488c4e08e4b)
![Case_Summary_2](https://github.com/user-attachments/assets/e1cc258d-a7f1-4490-b372-be03a807e9ab)

**Comment by alexbexe** [comment]
I will check first how your spreadsheet soft is loading the "symbol for decimal separation". Note that FloatingInfo uses "." for decimal separation.. but perhaps your PC is not configured and is reading in the wrong way.

Regards
